docs/Configuration.md

```diff
@@ -0,0 +1,119 @@
+# Configuring Just Saying
+
+## The publish subscribe pattern
```

Comment:
_"The publish-subscribe pattern"_? (dash added)

Reply:
done